content/app/github/wrappers/expandable-line-element-wrapper.js

```diff
@@ -0,0 +1,69 @@
+define("content/app/github/wrappers/expandable-line-element-wrapper", function(require, exports, module) {
+
+	// imports
+	var AbstractLineElementWrapper = require("content/app/github/wrappers/abstract-line-element-wrapper");
+
+	/**
+	 * Wrapper for expandable line DOM element.
+	 */
+	var ExpandableLineElementWrapper = Class(AbstractLineElementWrapper, {
+
+		$statics : {
+
+			CLASS_NAME : 'js-expandable-line',
+
+		},
+
+		/**
+		 * Constructor.
+		 * @param  {Element} domElement
+		 */
+		constructor : function (domElement) {
+			ExpandableLineElementWrapper.$super.call(this, domElement);
+			this._oldNumberRange = null;
+			this._newNumberRange = null;
+			this._proccessElement();
+		},
+
+		/**
+		 * Get old code line number range.
+		 * @type {Number[]}
+		 */
+		oldNumberRange : {
+			get : function () {
+				return this._oldNumberRange;
+			},
+			set : function (value) {
+				this._oldNumberRange = value;
+			},
+		},
+
+		/**
+		 * Get new code line number range.
+		 * @type {Number[]}
+		 */
+		newNumberRange : {
+			get : function () {
+				return this._newNumberRange;
+			},
+			set : function (value) {
+				this._newNumberRange = value;
+			},
+		},
+
+		/**
+		 * Override. Check if current line contains or represents specified line.
+		 * @param {Number} 		lineNumber
+		 * @return {Boolean}
+		 */
+		containsLine : function (lineNumber) {
```

Comment:
I forgot to mention, maybe it would be better that we offer an option that user can jump to old line number. For example, we could have two input fields (one for old ln, one for new ln) which could be easily switched via Tab button. 

That is overkill for now, but I just wanted to mention so that you could process the idea.